validation/tmSchemaGenerator.js

```diff
@@ -57,7 +57,7 @@ let tdSchema = JSON.parse(fs.readFileSync('validation/td-json-schema-validation.
 // do all the manipulation in order
 let tmSchema = staticReplace(tdSchema)
 tmSchema = removeRequired(tmSchema)
-tmSchema = removeEnum(tmSchema)
+// tmSchema = removeEnum(tmSchema)
```

Comment:
not sure why this is commented out

Reply:
I commented it out only for now as I wasn't sure if you want to keep this or some other kind of enum removal in the schema generation. I would propose keeping the enum values in but maybe this is something you want to discuss in the working group first.

Reply:
After thinking about it, one purpose of the enum removal is probably to enable the use of placeholders, I guess? Maybe the enums could be turned into an `anyOf` instead that additionally contains a regex item with a pattern like `"{{PLACEHOLDER}}"`? (Or rather, more formal, `\{{2}[A-Za-z]+\}{2}`).

Reply:
> After thinking about it, one purpose of the enum removal is probably to enable the use of placeholders, I guess?

An example that illustrates the current problem would be the following: https://www.jsonschemavalidator.net/s/qFot0kvR

Reply:
> I will try my take on this and commit to this branch

Awesome, thank you!